### DISCUSSION TITLE
如果能接入豆包模型就好啦!

### DISCUSSION THREAD

**Discussion by LonePheasantWarrior** [zhayujie/chatgpt-on-wechat#2173] [Ideas]
[豆包的大语言模型](https://www.volcengine.com/docs/82379/1263512)很适合角色扮演,接到微信里一定很有趣!

超期待,大佬拜托了

**Comment by LonePheasantWarrior** [comment]
其支持[智能体作为模型调用](https://www.volcengine.com/docs/82379/1285207),角色扮演绝了

**Comment by cmgzn** [comment]
豆包支持openai接口，直接用openai的就可以

**Reply by LonePheasantWarrior** [reply]
豆包的bot接口跟标准openAI的稍有不同
我是微调了下moonshot的调用方式适配的
有段时间没关注了,不知道项目现在是否支持了直接接入bot

**Reply by xw19930** [reply]
大佬接入豆包 具体要怎么操作啊

**Reply by jonathanyb** [reply]
> 豆包的bot接口跟标准openAI的稍有不同 我是微调了下moonshot的调用方式适配的 有段时间没关注了,不知道项目现在是否支持了直接接入bot

看了最新更新，还是不支持直接接入

**Reply by jonathanyb** [reply]
> 豆包支持openai接口，直接用openai的就可以

接入url地址都不一样，肯定没法直接用，还是需要修改源码，但是修改源码以后没法再服务器上直接使用Docker镜像，需要自己打包，这个更麻烦

**Reply by cmgzn** [reply]
> > 豆包支持openai接口，直接用openai的就可以
> 
> 接入url地址都不一样，肯定没法直接用，还是需要修改源码，但是修改源码以后没法再服务器上直接使用Docker镜像，需要自己打包，这个更麻烦

额可以啊，我都用了一个多月了，你是火山方舟的豆包api吗https://www.volcengine.com/docs/82379/1330626

**Reply by cmgzn** [reply]
> 大佬接入豆包 具体要怎么操作啊

把openai接口的base url直接改成豆包的就可以https://www.volcengine.com/docs/82379/1330626

**Reply by cmgzn** [reply]
> 豆包的bot接口跟标准openAI的稍有不同 我是微调了下moonshot的调用方式适配的 有段时间没关注了,不知道项目现在是否支持了直接接入bot

确实是v3版本才支持

**Comment by jonathanyb** [comment]
确实很需要豆包的api接入，现在ai 用户数量排行榜中，豆包已经排到第二位了，仅次于GPT，再多一年豆包可能会超越gpt变成用户量最多的ai
